### ISSUE TITLE
Nalu build error due to Trilinos includes

### ISSUE THREAD

**Issue by spdomin** [trilinos/Trilinos#703]
10/13/2016 NaluRtest
PROCESS FAILED
NaluCFD/Nalu SHA1: 636a674
Trilinos SHA1: 46a01d623c988b686e9af23e2eda920a2aa51de8

/scratch/spdomin/nightlyBuildAndTest/Nalu/include/LinearSolverTypes.h:13,
                 from /scratch/spdomin/nightlyBuildAndTest/Nalu/include/LinearSolver.h:12,
                 from /scratch/spdomin/nightlyBuildAndTest/Nalu/src/LinearSolvers.C:10:
/home/spdomin/gitHubWork/scratch_build/install/gcc4.8.5/Trilinos_nightly_release/include/Kokkos_Sparse_impl_spmv.hpp:53:43: fatal error: TpetraKernels_ETIHelperMacros.h: No such file or directory
 #include "TpetraKernels_ETIHelperMacros.h"

**Comment by bmpersc** [comment]
This appears to be an issue with the installation of that file. I am working on a fix for it now.

**Comment by bmpersc** [comment]
I just pushed a fix to develop. The missing file is now being installed properly.

**Comment by spdomin** [comment]
Could you please augment the Trilinos policy to wait until user verification before issues are closed?

**Comment by maherou** [comment]
This bug should still be open until confirmed by reporter.

**Comment by ibaned** [comment]
Also affects Albany. Should be okay once `develop` is merged into `master`.

**Comment by spdomin** [comment]
My nightly process for develop passed this morning. Feel free to close the issue, or we can wait until it is confirmed in master - whatever fits the Trilinos SQA process.
